pages/get-started.tsx

```diff
@@ -142,13 +142,14 @@ export default function GetStartedPage() {
           title={content.steps[2].title}
           bodyText={content.steps[2].body}
         >
-          <Grid container columnSpacing={8} sx={{ my: 8 }}>
+          <Grid container spacing={5} sx={{ mb: 8, mt: 0 }}>
```

Comment:
Not sure why but this mt:0 seems to be required because without it, this grid doesn't detect the text above it and it measures the distance from the middle of the above text. It may be related to markdown component that creates an extra component as shown in the below image in the elements section.

<img width="1469" alt="image" src="https://github.com/clearviction-devs/clearviction-wa/assets/29575804/94508daa-9abb-4dba-b514-522d64d90f65">

Reply:
The PR is ready for review

Reply:
I did some tests to understand this, and I didnt realize the exact thing that create this behavior too :smile: